ArduPlane/servos.cpp

```diff
@@ -541,8 +541,14 @@ void Plane::set_servos_controlled(void)
             // STABILIZE mode with THR_PASS_STAB set
             SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, get_throttle_input(true));
         } else {
-            SRV_Channels::set_output_scaled(SRV_Channel::k_throttle,
+            if (g2.flight_options & FlightOptions::CENTER_THROTTLE_TRIM) {
```

Comment:
I would move this check, to the top of the new function.

Reply:
I am not sure I agree with this....from a readability/understandability point of view....since the call from here would be always to get_centered_throttle_input.....which would be misleading naming....guess I could rename the function, but to what? get_throttle is already taken....get_thr_in()?get_adjusted_throttle()?

Reply:
so renamed function and moved option check...retested in SITL